portal-api/api-tests/insertUtilisationReportDetails.js

```diff
@@ -4,13 +4,13 @@ const { DB_COLLECTIONS } = require('./fixtures/constants');
 const insertOneUtilisationReportDetails = async (report) => {
   console.info('inserting one utilisation report details');
   const collection = await db.getCollection(DB_COLLECTIONS.UTILISATION_REPORTS);
-  await collection.insertOne(report);
+  return collection.insertOne(report);
```

Comment:
I have a preference for explicitly awaiting in the low down functions like here so it's clear to the reader what the asynchronous part of the function is. In this case it makes no difference as it's the last line and there aren't any other processes happening after it but if anyone was to add anything it could be weird behaviour if they forget to put an await back in.

Aware in this case it's more of a style nitpick than a behaviour nitpick - if you have any reasons on why you prefer what you've done would be keen to hear them!